### PR TITLE
Apply a few suggestions

### PR DESCRIPTION
**Pull request by seratch**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

This pull request suggests a few minor things for the template.

If you are introducing a new sample app in this pull request, please also include a detailed description of the application’s purpose and functionality.

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019- Slack Technologies, LLC
+Copyright (c) 2022- Slack Technologies, LLC
```
Comment (seratch) [inline LICENSE:3]: This template started this year. Also, I was thinking of a better placeholder than the Slack company here as the app will be owned by external devs... but I could not come up with any great ideas.

File: README.md [hunk 2]
```diff
@@ -20,9 +20,7 @@ CLI.
 
 ## Setup
 
-Before getting started, make sure you have a development workspace where you
-have permissions to install apps. If you don’t have one set up, go ahead and
-[create one](https://slack.com/create).
+Before getting started, make sure you have a development workspace where you have permissions to install apps. If you don’t have one set up, go ahead and [create one](https://slack.com/create). Also, please note that the workspace requires any of [the Slack paid plans](https://slack.com/pricing).
```
Comment (seratch) [inline README.md:23]: We can emphasize paid workspaces are required even for development.

File: LICENSE (hunk 1 above)
Comment (e-zim) [reply LICENSE:3]: "Slack Technologies, LLC et al." could be a placeholder to complete the license while still leaving space for the external dev, but I'm not sure if this makes sense in a license..

**Review by misscoded — APPROVED** [review]
LGTM! Would you mind opening up PRs for these changes in the other `deno-*` templates?

- [deno-hello-world](https://github.com/slack-samples/deno-hello-world)
- [deno-blank-template](https://github.com/slack-samples/deno-blank-template)
- [deno-github-functions](https://github.com/slack-samples/deno-github-functions)
- [deno-request-time-off](https://github.com/slack-samples/deno-request-time-off)

**Comment by seratch** [comment]
@misscoded @e-zim Thanks for your reviews! I will make the same changes to the other repos later on.

**Pull request merged**